docs/first-order/receptive_fields.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<p>can you add axis size on one of the plots</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>